### DISCUSSION TITLE
Ollama Error

### DISCUSSION THREAD

**Discussion by VinojRaj** [langflow-ai/langflow#1804] [Q&A]
I am new to Langflow and I was trying to use Llama2 through Ollama as the model but I am getting the following error:
ValueError: Error building vertex Ollama: ChatOllamaComponent.build() missing 1 required positional argument: 'base_url'

The base url is default on http://localhost:11434/

**Comment by adnanbwp** [comment]
I am having the same issue. 

I have checked that Ollama is working by opening http://127.0.0.1:11434/ in my browser. It says 'Ollama is running'.

Using Ollama LLM gives the (missing 1 required positional argument: 'base_url') error. 

No errors if I use OpenAI. 

I have only 1 model which is (llama3).

**Comment by Cristhianzl** [comment]
Hi! 👋

We are using other channels as our official means of communication with users. We apologize for the delayed response.
For faster assistance, please direct your questions to our Discord channel or open a GitHub issue, as we are much more active on these platforms.

[Langflow Discord Server](https://discord.com/invite/EqksyE2EX9)

Thank you for your understanding.

Best regards,
The Langflow Team
